docs/molgenis/use_permissions.md

```diff
@@ -23,4 +23,25 @@ In addition there are two special users:
 ## Users can be managed by admin
 
 Individuals can 'sign up' to register themselves to MOLGENIS databases, choosing user name and password. Special user is
-the 'admin'. Only this user can see and create other users.
\ No newline at end of file
+the 'admin'. Only this user can see and create other users.
+
+## Sign-in using Open ID Connect (OIDC)
+
+Users can be authenticated using an existing account via Open ID Connect (OIDC).
+To enable users to sign-in using OIDC, the Molgenis instance must be configured to use OIDC.
+When OIDC is enabled, users are no longer presented with the option to sign-up
+
+When OIDC is enabled, the admin user can bypass the oidc login by using the admin route (```[service-location]/apps/central/#/admin```)
+
+### Enabling OIDC
+To enable OIDC the following environment variables need to be set:
+```
```

Comment:
I suppose I would also like to be able to configure this as end user, and not only via environment. Because I might not have access to this when running on a cloud service. Is that possible?